test/fixtures/import_only_example/main.tf

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+resource "random_pet" "main" {
+  length    = 1
+  prefix    = "simple-example"
+  separator = "-"
+}
+
+module "example" {
+  source                        = "../../../examples/import_only_example"
```

Comment:
Generally we no longer need wrapping fixtures. In this case we can remove this and directly execute example in the test.

Reply:
Fixture removed!